obspy/io/sac/arrayio.py

```diff
@@ -462,7 +462,7 @@ def header_arrays_to_dict(hf, hi, hs, nulls=False, encoding='ASCII'):
                  if val != HD.FNULL] + \
                 [(key, val) for (key, val) in zip(HD.INTHDRS, hi)
                  if val != HD.INULL] + \
-                [(key, val.decode(encoding)) for (key, val)
+                [(key, val.decode(encoding).strip()) for (key, val)
```

Comment:
Why strip? Could be intentional non-space whitespace, I guess? E.g. storing Component code after two spaces.. dunno. Very edge case-y I guess.

Reply:
Hi all.  I think I used `.strip` in [`sactrace.py`](https://github.com/obspy/obspy/blob/master/obspy/io/sac/sactrace.py#L653) because it felt more natural to use strings with no spaces for `SACTrace` class attributes.  When string headers are put back into NumPy Land for writing to file, they get padded by NumPy anyway, so I'm ambivalent this change.  If I passes tests, it seems OK to proceed.